test/script/run_tests.sh

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env bash
 
-set -eo pipefail # abort if any of the commands exit badly
+if [ "$TRACE" != "" ]; then
+    export PS4='${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+    set -o xtrace
+fi
+set -o errexit
+set -o pipefail
```

Comment:
Reviewer note: This is bash boilerplate that I typically use in bash files (including some already in this repo). It adds support for `TRACE=1 .../run_tests.sh` to get xtrace log output.